examples/scene/prefab.rs

```diff
@@ -0,0 +1,205 @@
+use std::{fs::File, io::Write};
+use bevy::{ecs::schedule::ShouldRun, prelude::*, reflect::TypeRegistry, scene::SceneSpawnError, utils::HashMap};
+
+/// Basic Prefab support.
+/// To use the factory, spawn a component using Commands or World, and the Handle<DynamicScene> of the prefab.
+/// Then add to the factory queue. When the scene is loaded, 
+/// the components of the first entity listed in the file will be added to the entity.
+/// 
+/// LIMITATIONS
+///     Component data from the prefab won't be available immediately; the PrefabFactory system runs once per frame,
+///     and the actual scene must load before the scene is applied.
+///     
+///     Only the FIRST entity listed in the file will be inserted into the provided entity.
```

Comment:
```suggestion
///     Only the components from the FIRST entity listed in the file will be inserted into the provided entity.
```

Hopefully this is what you meant :)